packages/utilities/src/lib/tryParseJSON.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Try parse a stringified JSON string.
+ * @param value The value to parse
+ */
+export function tryParseJSON(value: string): object | string {
```

Comment:
Should return whatever json.parse says it returns in typings

Reply:
`JSON.parse` return type is `any`